src/SFA.DAS.Forecasting.Application/Estimations/Validation/AddApprenticeshipValidator.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
```

Comment:
This class is very big and does a lot of validation.  I prefer it if each validation rule was placed into it's own ValidationRule class with a common interface for all.